cluster/addons/fluentd-gcp/scaler-policy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: scalingpolicies.scalingpolicy.kope.io
```

Comment:
`kope.io`?

Reply:
I just reused the ScalingPolicy defined by @justinsb in https://github.com/justinsb/scaler. I hope we can ultimately migrate to autoscaling controlled by this ScalingPolicy in the long term.